### ISSUE TITLE
Cannot input chinese. LookupError: unknown encoding: UTF-16BE

### ISSUE THREAD

**Issue by CY-Qiu** [PyFPDF/fpdf2#322]
Describe the bug

  File "fpdf/fpdf.py", line 220, in wrapper
  File "fpdf/fpdf.py", line 1727, in text
LookupError: unknown encoding: UTF-16BE

**Minimal code**
```python
from fpdf import FPDF

if __name__ == "__main__":
    # Setup
    pdf = FPDF("P", "mm", "A4")
    pdf.add_page()

    pdf.add_font("simsun", fname="/usr/share/fonts/simsun.ttf", uni=True)
    pdf.set_font("simsun", size=8)
    pdf.write(8, "你好")
    # Save result
    pdf.output("test.pdf")

```

**Environment**
* Kylin Linux arm64
* Python 3.7.9
* fpdf2 2.4.6

I guess it is possible with the big endian system. Refer:
https://github.com/pypa/pip/issues/6054
Maybe we should change the encoding "UTF16-BE" to "utf-16-be"？

**Comment by Lucas-C** [comment]
Hi @CY-Qiu!

Thank you for reporting this.

"UTF16-BE" & "utf-16-be" are the same.

I tested your code, and it works fine on my system!

I used this font file: https://github.com/micmro/Stylify-Me/blob/master/.fonts/SimSun.ttf

This is quite strange...

**Comment by Lucas-C** [comment]
I don't really see why you are missing this encoding on your system, it should be standard:
https://docs.python.org/3/library/codecs.html#standard-encodings

Could you try replacing `UTF-16BE` in `fpdf/fpdf.py` by `utf-16-be` and see if that solves your issue?

**Comment by CY-Qiu** [comment]
After change `UTF16-BE` to `utf-16-be`, my problem solved.
In addition, I tried to use `UTF16-BE`  directly in python command line. Here is the result.
```
>>> haha = "123"
>>> haha.encode("UTF16-BE")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
LookupError: unknown encoding: UTF16-BE
>>> haha.encode("utf-16-be")
b'\x001\x002\x003'
```

**Comment by Lucas-C** [comment]
I made the switch in https://github.com/PyFPDF/fpdf2/pull/326

@CY-Qiu could you check if your issue is solved in the latest `fpdf2` version from git please ?

    pip install git+https://github.com/PyFPDF/fpdf2.git@master

**Comment by CY-Qiu** [comment]
@Lucas-C I can confirm the newest master branch work fine now. Thank you!

**Comment by Lucas-C** [comment]
Great! This is as now been released as part of `v2.5.0`
